src/response/chi0.jl

```diff
@@ -295,6 +295,7 @@ end
 
     # compute δψnk band per band
     δψ = zero.(ψ)
+    Hψ_extra = ham * ψ_extra
```

Comment:
I think I would do this inside the `ik` loop k-point by k-point to save some memory.